### PR TITLE
Handle archives.relacements deprecation in goreleaser

### PR DESCRIPTION
**Pull request by hdm23061993**
## Fixes Issue 
#164 

## Changes proposed

- Handle `archives.relacements` deprecation in goreleaser
Resource : [Deprecation Notice](https://goreleaser.com/deprecations/#archivesreplacements)

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by hdm23061993** [comment]
As per [official docs](https://goreleaser.com/customization/archive/), a default `name_template` is used if `format: tar.gz`

``` yaml
# .goreleaser.yaml
    # Archive name template.
    # Defaults:
    # - if format is `tar.gz`, `tar.xz`, `gz` or `zip`:
    #   - `{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ with .Arm }}v{{ . }}{{ end }}{{ with .Mips }}_{{ . }}{{ end }}{{ if not (eq .Amd64 "v1") }}{{ .Amd64 }}{{ end }}`
    # - if format is `binary`:
    #   - `{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ with .Arm }}v{{ . }}{{ end }}{{ with .Mips }}_{{ . }}{{ end }}{{ if not (eq .Amd64 "v1") }}{{ .Amd64 }}{{ end }}`
    name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
```


## Screenshot

image

**Comment by undera** [comment]
This will probably break the Helm plugin install process, since it relies on these namings.

Looks like we'll have to leave the namings as-is for now, otherwise we'll break the ability for users to install arbitrary versions. The way Helm plugins work is it downloads the latest repo code and then uses the shell script (https://github.com/komodorio/helm-dashboard/blob/main/scripts/install_plugin.sh) to run the install. So if the version of script is arranged for the new naming, we will lose the old version installation capability. Complicating install script because of that is not worth it.

Bottom line: I think the small warning in CI is bearable as opposed to real disruption and risks of install process.

**Comment by hdm23061993** [comment]
I handled the case in `install_plugin.sh` using a simple `if-else`
In [chartmuseum/helm-push](https://github.com/chartmuseum/helm-push/blob/main/scripts/install_plugin.sh) as well, I see they have moved from `x86_64` to `amd64`, as handled by me in this change

https://github.com/chartmuseum/helm-push/blob/main/scripts/install_plugin.sh#L15-L22

**Comment by undera** [comment]
This would require some extended testing:

1. To see on forked repo that release generates file names successfully
2. To test that new install script takes those new filenames correctly
3. To test that new install script still allows installing previous version

Did you test all of above cases @hdm23061993 ?

**Comment by hdm23061993** [comment]
Will check on it

**Comment by hdm23061993** [comment]
Will pick later
Closing as of now
